triage/summarize.py

```diff
@@ -87,8 +88,8 @@ def repl(m):
         # for long strings, remove repeated lines!
         s = re.sub(r'(?m)^(.*\n)\1+', r'\1', s)
 
-    if len(s) > 200000:  # ridiculously long test output
-        s = s[:100000] + '\n...[truncated]...\n' + s[-100000:]
+    if len(s) > 10000:  # ridiculously long test output
```

Comment:
10k doesn't seem that long for a kubernetes job :^)
do we know how many jobs this likely affects?

Reply:
this is fair, but I'm not sure how much additional signal it gives for clustering, I'll see if I can figure what the impact is

Reply:
It looks like this truncates 440 more test failures than before, out of ~98K.  Horrible jq and awk were involved.
```
498 tests with failure_text > 10000 vs 97359 with less
58 tests with failure_text > 200000 vs 97799 with less
```

Reply:
this is fine